Main.hs

```diff
@@ -51,11 +51,22 @@ main = do
             case stripPrefix dir file of
               Nothing -> Left file
               Just x -> case splitDirectories x of
-                ("/" : "build" : hostOs : ghcVersion : packageName : componentType : subComponent : "build" : modulePath) -> Right GhcFile{..}
+                ("/" : "build" : hostOs : ghcVersion : packageName : componentType : subComponent : "build" : modulePath) ->
+                  let srcFilePath = file
```

Comment:
let's write `= partitionEithers $ files <&> \srcFilePath ->` in line 50 in order to avoid repetition?

Reply:
Fixed.